src/xdp.props

```diff
@@ -7,7 +7,7 @@
     <!-- Project-wide properties -->
     <EbpfPackagePath>$(SolutionDir)packages\eBPF-for-Windows.0.15.1\</EbpfPackagePath>
     <WiXPackagePath>$(SolutionDir)packages\wix.3.14.1\</WiXPackagePath>
-    <FnPackagePath>$(SolutionDir)artifacts\fn\devkit-0.4.4\</FnPackagePath>
+    <WntPackagePath>$(SolutionDir)packages\win-net-test.0.5.1\</WntPackagePath>
```

Comment:
Could you add nuget publishing documentation in the https://github.com/microsoft/win-net-test repo? I see the nuget is now being built automatically, but I'm not sure how I'd publish a new version.